agent/task.go

```diff
@@ -146,26 +150,30 @@ func (a *Agent) runPreTaskCommands(ctx context.Context, tc *taskContext) {
 		taskGroup, err := model.GetTaskGroup(tc.taskGroup, tc.taskConfig)
```

Comment:
this isn't your code but this is TERRIFYING because it looks like the agent is doing a database query.

Reply:
What database query are you referring to? From what I see GetTaskGroup finds the task group from the version config.

Reply:
There isn't one. I don't think anything needs to change for this - we have other project parsing things in the model package which is fine